build.sbt

```diff
@@ -55,7 +57,11 @@ lazy val commonJsSettings = Seq(
     s"-P:scalajs:mapSourceURI:$a->$g/"
   },
   scalaJSStage in Global := FastOptStage,
-  parallelExecution := false
+  parallelExecution := false,
+  // Only use for scala.js for now
+  botBuild := sys.props.getOrElse("botBuild", default="false") == "true",
```

Comment:
This is probably me being paranoid, but since this is a global environment var should we namespace it a bit? Maybe `CATS_BOT_BUILD` or something?